requirements.txt

```diff
@@ -1,6 +1,6 @@
 vcsinfo>=2.1.105
 PyYAML==6.0
-requests==2.25.0
+requests>=2.25.0
```

Comment:
We need to move to requirements.in and the "resolved" requirements.txt. But this can be done in another PR later.

Reply:
Created issue for this -> https://github.com/adobe/buildrunner/issues/30